Update abbreviations.yml

xendez opened the pull request:
These are the most common abbreviations in Portugal and some in Brazil and other Portuguese speaking countries. If these regexp are too "expensive" please let me know and I will try to keep them more simple.

ENT8R commented:
Would it maybe be possible that you write/translate the comments to English?

westnordost commented:
The comment is only a translation of the comment in the English abbreviations.yml, so that is fine.
They are also not expensive.

Thank you for the contribution! The list is used in the street name input. Basically, these regex rules are automatically applied when finishing a word (it's **forced**, not like the usual word suggestions on Android). This is why the list should only include abbreviations which are clear and unambiguous to a Portuguese-speaking user. Is this the case with all the abbreviations you provided here? For English, it is not so clear, that is why I left out all those doctor, professor, ... etc there.

For example, is it clear that 
`Esc Eng Xendez`
means always
`Escadas Engenheiro Xendez`
?

If not, perhaps the list should be reduced a bit again (or commented out).

rugk commented:
For what are these abbrevations even used? Is not it only for street names? And do we really need to include stuff like "professor" there, considering whether/how often it appears in street names?

ENT8R commented:
@rugk, I think @westnordost already answered all your questions without even knowing them... :grinning:

rugk commented:
Well… the last one maybe not.

xendez commented:
Some questions were answered correctly by others, thank you.
@rugk, "professor" and "urb" are used in street names, but this time, in the second commit, I removed "urb" and others since in some cases it means another word (although lets say +-99% the time means "urbanização"). Generally "urbanização" is used in buildings but many streets have this word.

If this can be even more simple? Of course, but could be even more complex. Take a look at https://wiki.openstreetmap.org/wiki/Name_finder:Abbreviations#Portugu.C3.AAs_-_Portuguese (most of them was added by me). I've followed the advice, use the most common, with a few exceptions without false positives, and without false positives. Unfortunately some of the most used have false positives, so this time I've removed them.

Sorry if I've made something wrong, it's the first time I make a fork. As a non programmer, it's weird how GitHub works.

westnordost commented:
No, you did everything correctly. GitHub makes it a bit easier than usual with Git to make in-place "modification proposals" (Pull Requests), but of course it is still somewhat of a hurdle compared to simply editing a wiki.

I share with you the opinion that much of the metadata around OSM is something that may be better maintained in a wiki than in a code repository because many people who may help to collect and maintain this data may not be proficient with Git (in particular). This i why I find the execution of the [osm-community-index](https://github.com/osmlab/osm-community-index), being a GitHub repository rather than something more accessible, not ideal. (But the idea is awesome!)
I am currently collecting the metadata for the [default speed limits](https://wiki.openstreetmap.org/wiki/Default_speed_limits) for the different road and vehicle types for all the countries in the world. And for this, I chose to do it in the openstreetmap wiki and later write a parser for that instead of doing it in a Git repository because of the mentioned hurdle and of course of the higher visibility.

Aanyway, getting off topic here. So, after your changes, you would say this can be applied as it is now? As a native speaker, you are the specialist on that.

goldfndr commented:
Of the three *nossa senhora*, it looks to me like the middle is a subset of the last one. If yes then the middle can be eliminated.

westnordost commented:
You are right, I removed it.

xendez commented:
Thanks @goldfndr for the fix. I'm currently fixing translations and testing the last version 8.0 and it seems the Portuguese abbreviations are working.
